packages/components/src/Inset/Inset.tsx

```diff
@@ -1,16 +1,19 @@
 import { ReactNode } from 'react';
 import {
-  PaddingSpaceProp,
-  PaddingSpacePropX,
-  PaddingSpacePropY,
   cn,
   paddingSpace,
   paddingSpaceX,
   paddingSpaceY,
 } from '@marigold/system';
+import type {
+  PaddingSpaceProp,
+  PaddingSpacePropX,
+  PaddingSpacePropY,
+} from '@marigold/system';
+import type { AriaRegionProps } from '@marigold/types';
 
 export type InsetProps =
-  | {
+  | (AriaRegionProps & {
```

Comment:
why is there a and ?

Reply:
So that every of the following types also has the AriaRegionProps :)